setup.py

```diff
@@ -11,11 +11,18 @@
 with open('CHANGELOG.md') as changelog_file:
     changelog = changelog_file.read()
 
-requirements = []
-
-setup_requirements = ['pytest-runner', ]
-
-test_requirements = ['pytest', ]
```

Comment:
Why did you drop pytest here?

Reply:
We need these requirements only if we run tests like `python setup py test`. And we do not do this.